math/libfaiss/Portfile

```diff
@@ -1,62 +1,77 @@
+github.setup        facebookresearch faiss 1.5.3 v
 name                libfaiss
-version             20180214
-checksums           rmd160  e699f7ec89d8c6d5c4957022d26d162ab3b85d62 \
-                    sha256  6ef05ff911e68bb4ff3f209d2abb3a2e57d36f308176e1555266ecd055c565bf
+
 categories          math
-license             BSD
+license             MIT
 platforms           darwin
 maintainers         {@beauby fb.com:hoss} {@mdouze fb.com:matthijs} \
                     openmaintainer
 
-description         Efficient similarity search library from Facebook AI Research.
-long_description    Library for efficient similarity search and clustering of \
-                    dense vectors. It contains algorithms that search in sets \
-                    of vectors of any size, up to ones that possibly do not \
-                    fit in RAM. It also contains supporting code for \
-                    evaluation and parameter tuning. Faiss is written in C++ \
-                    with complete wrappers for Python/numpy. Some of the most \
-                    useful algorithms are implemented on the GPU.
+description         A library for efficient similarity search and clustering of dense vectors.
+
+long_description    Faiss is a library for efficient similarity search\
+                    and clustering of dense vectors. It contains\
+                    algorithms that search in sets of vectors of any\
+                    size, up to ones that possibly do not fit in\
+                    RAM. It also contains supporting code for\
+                    evaluation and parameter tuning. Faiss is written\
+                    in C++ with complete wrappers for\
+                    Python/numpy. Some of the most useful algorithms\
+                    are implemented on the GPU. It is developed by\
+                    Facebook AI Research.
+
+checksums           rmd160  3b1180dc7ceadc484bb86cfe32fb5cd1c4a694b9 \
+                    sha256  6962641a9183e12d394aa2f9c443593734a373d60c61fd0633edda5029d2fa78 \
+                    size    2762193
+
+depends_lib-append  port:libomp
+
+post-extract {
+    # fix destroot commands like `$(PYTHON) setup.py install`
+    reinplace -E \
+        "s|^(\[\[:space:]]+\\\$\\(PYTHON\\)\[\[:space:]]+setup\.py\[\[:space:]]+)(install)|\\1 --no-user-cfg \\2 ${destroot.destdir}|" \
+        ${worksrcpath}/python/Makefile
+}
 
 compilers.choose    cxx
-# The Apple native llvm compiler does not support OpenMP.
+# The Apple native llvm compiler does not support OpenMP
 compiler.blacklist  llvm-gcc-4.2
-compiler.fallback   macports-gcc-7
+compiler.fallback   macports-gcc-9
 compilers.setup
 
-use_configure       no
+configure.args-append \
+                    --without-cuda
+configure.cppflags-append \
+                    -I${prefix}/include/libomp
+configure.cflags-append \
+                    -fPIC -m64 -Wall -g -O3 -msse4 -mpopcnt -fopenmp \
+                    -Wno-sign-compare -std=c++11
```

Comment:
if you're adding ```c++11``` here shouldn't you make sure to get only a compiler that supports it (i.e., ```compiler.cxx_standard   2011```)? Besides that comment, I don't feel very qualified to review the changes to this port.

Reply:
Thanks. Superseded by https://github.com/macports/macports-ports/pull/7514 with your suggestions.